### DISCUSSION TITLE
Publish the app for winget-cli

### DISCUSSION THREAD

**Discussion by Morgy93** [spatie/ray#477] [Ideas]
Hello,

I'm currently trying to provide the manifest for the Windows Desktop app to winget-cli, but I'm in need for some additional data.

Maybe someone here could help me out: https://github.com/microsoft/winget-pkgs/issues/18093

**Comment by freekmurze** [comment]
This is the direct link to the latest version: https://ray-app.s3.eu-west-1.amazonaws.com/Ray%20Setup%201.14.7.exe

**Comment by Morgy93** [comment]
It's published! \o/

Happy `winget install ray` 😊
